html/canvas/offscreen/manual/draw-generic-family/2d.text.draw.generic.family.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<title>OffscreenCanvas test: 2d.text.draw.generic.family</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/html/canvas/resources/canvas-tests.js"></script>
+<script>
+function drawCanvas(ctx, family)
+{
+    ctx.font = '32px ' + family;
+    ctx.fillText(family, 0, 32);
+}
+
+function testDrawGenericFamily(family)
+{
+    var offscreenCanvas = new OffscreenCanvas(300, 150);
```

Comment:
Consider using `const` and `let` throughout.

Reply:
Okidokes.